conf/modules.config

```diff
@@ -543,7 +543,12 @@ process {
     }
 
     withName: TAXPASTA_MERGE {
-        ext.args =  { "-p ${meta.tool} -o ${meta.tool}_${meta.id}.${params.standardisation_taxpasta_format}" }
+        ext.args =  {
+            [
+                "-p ${meta.tool} -o ${meta.tool}_${meta.id}.${params.standardisation_taxpasta_format}",
+                params.taxpasta_taxonomy_files ? "--taxonomy" : "",
```

Comment:
Dn't forget to add `taxpasta_taoxnomy_files after `--taxonomy`!!

Reply:
Hmm not sure I understand what do you mean here.

Reply:
```suggestion
                params.taxpasta_taxonomy_files ? "--taxonomy ${params.taxpasta_taxonomy_files}" : "",
```